assets/scss/widgets.scss

```
@@ -0,0 +1,127 @@
+  text-decoration: underline;
+}
+
+.bsky-handle {
+  color: var(--color-text-muted);
+  font-size: 0.9rem;
+}
+
+.bsky-content {
+  margin: 0.5rem 0;
+  line-height: 1.5;
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)